### PR TITLE
Add manual testing instructions for Filter Events interface

### LINKED ISSUE #36074: Add testing instructions for Filter Events dialog
In manual testing for version 6.8 we felt that just using the [information documentation](https://docs.mantidproject.org/interfaces/utility/Filter%20Events.html) for the Filter Events dialog was not sufficient to help us understand how to test the dialog. Some step by step manual testing instructions, with expected results from tests performed, would be more helpful here.

### PR DESCRIPTION
**Pull request by adriazalvarez**
### Description of work
This PR adds manual testing instructions covering some basics of the GUI workflow and usages of the event filter interface. 
This has been raised repeatedly as an issue in testing (#36697 ,#36050). 

#### Summary of work

Fixes #36074. 

#### Further detail of work

Note that there are some issues with this interface that appear frequently during testing(#36073). I'll assign myself to work on these for next release, but I have added a 'pending note' at the beginning of the instructions. 
### To test:
- Build dev-docs.
- Testing instructions can be found in ``Documentation``->``Manual Testing``->``Filter Events Interface Testing``
- Follow test instructions.

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#. Back to the filter events interface, without changing anything else from previous interaction, go to ``Advanced Setup`` tab and tick the ``Fast Log`` checkbox. Go back to ``Filter by Time`` and click
+   on ``Filter`` button again. Workspace ending in ``_splitters`` should be a 2D workspace, all other workspaces should remain the same.
+#. Now on ``Algorithms`` select ``FilterEvents`` algorithm and click on ``Execute``. As ``InputWorkspace`` select ``FilteredTemp_0`` workspace. On Splitter workspace select ``FilteredTemp_0_splitters``,
+   and on ``Information Workspace`` select ``FilteredTemp_0_info``. Check on ``GroupWorkspaces`` checkbox. And name the ``OutputWorkspaceBaseName`` as ``FilteredTimeRedux``.
+#. ``Run`` the algorithm. On the ADS, a new group event workspace named ``FilteredTimeRedux`` should appear. The two event workspaces that it contains should be equal to the two event workspace previuosly generated
+   through the interface, ``FilteredTime_0`` and ``FilteredTime_1``.
```
Comment (jhaigh0) [inline dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:59]: I didn't get a group workspace here, I just got `FilteredTimeRedux_0` and `_1` output as separate workspaces. They were the same though.

**Review by jhaigh0 — CHANGES_REQUESTED** [review]
Good instructions, just a couple of comments / suggestions

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+--------------
+
+#. Open ``Interfaces`` > ``Utility`` > ``Filter Events``
+#. Browse for file 'CNCS_7860_Event.nxs' from the Usage Data set.
+#. Click ``Load``. After few seconds, plot should be updated with a graph of summed up Counts vs.Time.
+#. ``_Summed_..`` workspace should be generated on the ADS, as well as ``CNCS_7860_Event`` workspace.
+#. Clicking on ``Refresh`` button should update the drop-down list with the name of the loaded workspace.
+#. Check Vertical Range Sliders work properly. You can't overlap the sliders or cross the sliders.
+   Additionally, placing lower slider at the bottom of the range allows the other slider to move freely on the whole range.
```
[the remark this answers is not in the document]
Comment (adriazalvarez) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:30]: yeah, there is a little bit of padding. I think is coded on the sliders but I'm not sure. I'll investigate it tomorrow. 
Anyway I will rephrase it to avoid confusion with testers.

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#. Check Horizontal Range Sliders work properly. You can't overlap or cross the sliders.
+   Additionally, having left slider at the leftmost part of the range allows the other slider to move on the whole range.
+#. Check that the Text Edits for ``Starting Time`` and ``Stopping Time`` work correctly:
+
+   - Position of vertical bars on the plot are updated appropiately when changing the numerical value of the edits and clicking ``Set``.
+   - Setting a value on ``Starting Time`` larger than the current value on ``Stopping Time`` is not allowed, and viceversa.
```
[the remark this answers is not in the document]
Comment (adriazalvarez) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:36]: Yeah, it doesn't happen with every pair of values. so it may be some kind of rounding error. I will open an issue, there are many issues with this interface. I plan to work on it on next release.

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst (hunk 1 above)
Comment (adriazalvarez) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:59]: Mmm.. I'm getting the workspaces on a group. Did you tick on the `Group Workspaces` checkbox in the algorithm?

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jhaigh0) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:36]: Sounds like a good idea 👍

File: dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst (hunk 1 above)
Comment (jhaigh0) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:59]: Ah no, I don't think I did.

```suggestion
#. ``Run`` the algorithm with ``Group Workspaces`` checked. On the ADS, a new group event workspace named ``FilteredTimeRedux`` should appear. The two event workspaces that it contains should be equal to the two event workspace previuosly generated
   through the interface, ``FilteredTime_0`` and ``FilteredTime_1``.
```

Comment (adriazalvarez) [reply dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst:59]: It's kind of already indicated in previous instruction?

**Comment by adriazalvarez** [comment]
I added some of your suggestions and stripped down some lines that make part of issues on this interface. hopefully by next release these documents will be updated once interface is improved.

**Review by jhaigh0 — APPROVED** [review]
thank you for the changes
